Makefile

```diff
@@ -168,6 +168,11 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
+ginkgo:
+ifeq (, $(shell which ginkgo))
+	go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.5.0
```

Comment:
why the mod=mod?

Reply:
removed. we can keep go test for now I believe.
```
While you can run go test instead of the ginkgo CLI, Ginkgo has several capabilities that can only be accessed via ginkgo. We generally recommend users embrace the ginkgo CLI and treat it as a first-class member of their testing toolchain.
```

Reply:
my question was only about the need for -mod=mod

Reply:
Go didn't let me install it without it.
Are you ok with keeping the go test command?